src/main/scala/fs2/aws/internal/Internal.scala

```diff
@@ -41,11 +46,17 @@ object Internal {
   private[aws] case class MultiPartUploadInfo(uploadId: String, partETags: List[PartETag])
 
   private[aws] trait KinesisProducerClient[F[_]] {
-    implicit def byteList2ByteBuffer(l: List[Byte]): ByteBuffer = ByteBuffer.wrap(l.toArray)
 
-    private lazy val client = new KinesisProducer
+    val credentials = new DefaultAWSCredentialsProviderChain()
+    val region      = "us-east-1"
```

Comment:
Do we have to give it a default region ?, + I'm not sure this is a default, looks like even if region can be fetched from credentials / environment variable you're overriding it with us-east-1

I'm just thinking if we let the AWSCredentialsProvider do its job we won't have to override credentials everywhere in the code

Reply:
oooh, great point. it defaults to the region the ec2-instance resides in, so this wouldn't be good at all. I'll change